activities/Measure.activity/js/activity.js

```diff
@@ -0,0 +1,438 @@
+			document.getElementById("timebased2").style.width="0px";
+			document.getElementById("timebased2").style.height="0px";
+			document.getElementById("timebased2").style.visibility="hidden";
+
+			document.getElementById("freqbased2").style.width="47px";
+			document.getElementById("freqbased2").style.height="47px";
+			document.getElementById("freqbased2").style.left="-20px";
+			document.getElementById("freqbased2").style.visibility="visible";
+
+			document.getElementById("timepitchvalue").min="1.3";
+			document.getElementById("timepitchvalue").max="6.3";
+			document.getElementById("timeratevalue").min="1";
+			document.getElementById("timeratevalue").max="5";
+			document.getElementById("timeratevalue").value=retreived_datastore.timeyp;
+			document.getElementById("timepitchvalue").value=retreived_datastore.timexp;
+
+			retreived_datastore.time_flag=1;
+			retreived_datastore.freq_flag=0;
+			retreived_datastore.last_graph=1;
+			
+		});
+	
+		document.getElementById("freqbased2").addEventListener('click',function(){
+			document.getElementById("one").style.display="none";
+			document.getElementById("two").style.display="block";
+			document.getElementById("freqbased2").style.visibility="hidden";
+			document.getElementById("timebased2").style.width="47px";
+			document.getElementById("timebased2").style.height="47px";
+
+			document.getElementById("freqbased2").style.width="0px";
+			document.getElementById("freqbased2").style.height="0px";
+			document.getElementById("timebased2").style.visibility="visible";
+
+			document.getElementById("timepitchvalue").min="1";
+			document.getElementById("timepitchvalue").max="2";
+			document.getElementById("timeratevalue").min="0.2";
+			document.getElementById("timeratevalue").max="2";
+			document.getElementById("timepitchvalue").value=retreived_datastore.freqxp;
+			document.getElementById("timeratevalue").value=retreived_datastore.freqyp;
+
+			retreived_datastore.freq_flag=1;
+			retreived_datastore.time_flag=0;
+			retreived_datastore.last_graph=0;
+		});
+
+		document.getElementById("fullscreen-button").addEventListener('click', function(event) {
+			document.getElementById("main-toolbar").style.opacity = 0;
+			document.getElementById("canvas").style.top = "0px";
+			document.getElementById("unfullscreen-button").style.visibility = "visible";
+		});
+
+		//Return to normal size
+		document.getElementById("unfullscreen-button").addEventListener('click', function(event) {
+			document.getElementById("main-toolbar").style.opacity = 1;
+			document.getElementById("canvas").style.top = "55px";
+			document.getElementById("unfullscreen-button").style.visibility = "hidden";
+		});
+	
+		function timebased(argument_values){
+			if(document.getElementById("timebased").value == 0){
+				var s = function(p){
+					p.mic1;
+					p.yslider;
+					p.xslider;
+					p.buttonx1;
+					p.buttony1;
+					p.buttonx2;
+					p.buttony2;
+					p.bg;
+					p.img;
```

Comment:
Not sure to understand what is done here?

Reply:
It was for testing purposed . Fixed